projects/packages/my-jetpack/_inc/components/onboarding-screen/styles.module.scss

```diff
@@ -1,7 +1,11 @@
 :global(body) {
 	font-family: "SF Pro Text", -apple-system, BlinkMacSystemFont, "Avenir Next", "Avenir", "Segoe UI", "Helvetica Neue", "Helvetica", "Cantarell", "Ubuntu", "Roboto", "Noto Sans", "Arial", sans-serif;
```

Comment:
I think in the other PR @keoshi mentioned to use different font stack

Reply:
It's exactly the same font stack, but with SF Pro Text as the main one (as it is in the reference)